### DISCUSSION TITLE
[Feature Request] useFocusTrap()

### DISCUSSION THREAD

**Discussion by TomSmedley** [juliencrn/usehooks-ts#415] [Ideas]
This would be helpful for building accessible modals. 

(https://www.npmjs.com/package/react-focus-lock)

**Comment by andreasvirkus** [comment]
@juliencrn any plans for implementing this? would you be open to contributions? I see a ton of open PRs so not sure if you're actively tackling topics?

I'd propose adding https://www.npmjs.com/package/focus-trap (their React wrapper: https://github.com/focus-trap/focus-trap-react)
